get_CLN_data.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+chain=$1
+color_green=$2
+color_red=$3
+ln_git_version=$4
```

Comment:
$ln_git_version uses 

_<sub>_if [ -z "$ln_git_version" ]; then_
   _ln_git_version=$(curl -s https://api.github.com/repos/lightningnetwork/lnd/releases/latest | grep -oP '"tag_name": "\K(.*)(?=")')_
fi</sub>_

Makes no sense to check LND version against CLN version.

Reply:
Good catch, thanks!
Fixed by:
https://github.com/raspibolt/raspibolt-pulse/pull/2/commits/9cbcd26a37b8fcec56c02c3084f6a54508164544